frontend/app/services/TouchpointBackend.scala

```diff
@@ -70,6 +72,15 @@ case class TouchpointBackend(
       _ <- subscriptionService.downgradeSubscription(member, FriendTierPlan)
     } yield {
       memberRepository.metrics.putDowngrade(member.tier)
+      track(
+        MemberActivity(
+          "downgradeMembership",
+          MemberData(
+          member.salesforceContactId,
```

Comment:
Maybe we can just pass `member` in here, and add another `apply` method on `MemberData` to extract the correct data ?